news/2017-08-10-new-featured-artist-wisp-x.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "New Featured Artist: Wisp X"
+date: 2017-08-11 03:20:00 +0000
+---
+
+Energetic, emotional and enthralling, meet [**Wisp X**](https://osu.ppy.sh/beatmaps/artists/16), our newest addition to the osu! Featured Artist roster!
+
+[![](https://assets.ppy.sh/artists/16/header.jpg?201708102)](https://osu.ppy.sh/beatmaps/artists/16)
+
+A fresh new take on some classic EDM and electronica staples, **Wisp X**'s sound has been making waves lately, having been featured on the popular *Waifu Wednesdays* promotional channel.
+
+Give one of their latest tracks, *Eventide* a listen and see for yourself:
+
+<iframe style="border: 0; width: 100%; height: 120px;" src="https://bandcamp.com/EmbeddedPlayer/track=323729686/size=large/bgcol=ffffff/linkcol=0687f5/tracklist=false/artwork=small/transparent=true/" seamless><a href="http://wispx.bandcamp.com/track/eventide">Eventide by Wisp X</a></iframe>
+
+
+A multitude of styles and sounds reign supreme in **Wisp X**'s work, with tracks that range from raise-the-roof dancefloor hits to pensive and emotional journeys. We may say this every time, but there's absolutely bound to be something for everyone in **Wisp X**'s work.
+
+Don't believe it? Give *Mint* a listen, and experience a taste of their talent:
+
+<iframe style="border: 0; width: 100%; height: 120px;" src="https://bandcamp.com/EmbeddedPlayer/track=2640945812/size=large/bgcol=ffffff/linkcol=0687f5/tracklist=false/artwork=small/transparent=true/" seamless><a href="http://wispx.bandcamp.com/track/mint">Mint by Wisp X</a></iframe>
+
+
+We're proud to present a whopping **24** tracks from **Wisp X** for usage in osu! mapping, available now at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/16). 
+
+As always, any track present on a Featured Artist listing comes pre-timed for your convenience, and is 100% licensed for use for mapping in osu!, so worry less, and map more!
+
+We're looking forward to exploring **Wisp X**'s sound as it continues to grow and evolve in the future, and we hope you are too! Like what you hear? Make sure to check our their [bandcamp](https://wispx.bandcamp.com/) and follow them on [twitter](https://twitter.com/wispx) to catch their latest work!
```

Comment:
typo: `Make sure to check our their ` -> `Make sure to check out their `
(change `our their` to `out their`)